app/src/components/Map/ControlPanel.jsx

```diff
@@ -32,30 +32,52 @@ class ControlPanel extends Component {
     }
   }
 
+  calculateVesselCount() {
+    const min = this.props.timelineInnerExtent[0].getTime();
+    const max = this.props.timelineInnerExtent[1].getTime();
+
+    if (this.props.chartData.length === 0) {
+      return '-';
+    }
+
+    let result = this.props.chartData.reduce((acc, elem) => {
+      if (elem.date >= min && elem.date <= max) {
+        return acc + elem.value;
```

Comment:
This belongs to a reducer. For instance listen to SET_INNER_TIMELINE_DATES in the timebar reducer.